src/ng/browser.js

```diff
@@ -79,6 +83,15 @@ function Browser(window, document, $log, $sniffer) {
     }
   };
 
+  self.shutdown = function() {
+    active = false;
+    forEach(currentIntervalIds, function(ignore, intervalId) {
+      delete currentIntervalIds[intervalId];
+      clearInterval(+intervalId);
+    });
+    jqLite(window).off('hashchange popstate', cacheStateAndFireUrlChange);
+  };
```

Comment:
I know it was done before either, but wouldn't it make sense to clear timeouts as well?